app/config/settings.py

```diff
@@ -789,13 +789,30 @@ def get_private_ip():
     "pymdownx.magiclink",
     "pymdownx.tasklist",
     "pymdownx.tilde",
-    BS4Extension(),
 ]
-MARKDOWN_POST_PROCESSORS = []
+MARKDOWN_POST_PROCESSORS = [
+    ExtendHTMLTagClasses(
+        {
+            "img": ["img-fluid"],
+            "blockquote": ["blockquote"],
+            "table": [
+                "table",
+                "table-hover",
+                "table-borderless",
+            ],
+            "thead": ["thead-light"],
+            "code": ["codehilite"],
+        }
```

Comment:
This class is only used once and the configuration is static, it would be better if this configuration is just done on the class once so that the logic is not split up in multiple places unnecessarily.